<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="::Qters::QrWidgets::QrChatMessage" name="bubble1" native="true"/>
   </item>
   <item>
    <widget class="::Qters::QrWidgets::QrChatMessage" name="bubble2" native="true"/>
   </item>
   <item>
    <widget class="::Qters::QrWidgets::QrChatMessage" name="bubble3" native="true"/>
   </item>
   <item>
    <widget class="::Qters::QrWidgets::QrChatMessage" name="bubble4" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>::Qters::QrWidgets::QrChatMessage</class>
   <extends>QWidget</extends>
   <header>chat/qrchatmessage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
